src/main/java/in/zapr/druid/druidry/virtualColumn/ExpressionVirtualColumn.java

```diff
@@ -0,0 +1,28 @@
+package in.zapr.druid.druidry.virtualColumn;
+
+import javax.annotation.Nonnull;
+
+import in.zapr.druid.druidry.dimension.enums.OutputType;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NonNull;
+
+@Getter
+@EqualsAndHashCode(callSuper = true)
+public class ExpressionVirtualColumn extends DruidVirtualColumn {
+    private static final String EXPRESSION_VIRTUAL_COLUMN = "expression";
+
+    @NonNull
+    private String expression;
+
+    public ExpressionVirtualColumn(@NonNull String name, @Nonnull String expression) {
```

Comment:
You can remove this constructor and annotate below constructor with builder. No need to set default value of `OutputType` druid will take care of that if we don't send this parameter in query.